### ISSUE TITLE
Missing alt attribute on images

### ISSUE THREAD

**Issue by linkstrifer** [brianeto/link-tutorial#4]
Every image should have an alt attribute for accessibility reasons:

https://github.com/brianeto/link-tutorial/blob/226d8c21bce472dc4aee8a7e09a05210cb491f9b/index.html#L29

https://github.com/brianeto/link-tutorial/blob/226d8c21bce472dc4aee8a7e09a05210cb491f9b/index.html#L42

https://github.com/brianeto/link-tutorial/blob/226d8c21bce472dc4aee8a7e09a05210cb491f9b/index.html#L47

https://github.com/brianeto/link-tutorial/blob/226d8c21bce472dc4aee8a7e09a05210cb491f9b/index.html#L54

https://github.com/brianeto/link-tutorial/blob/226d8c21bce472dc4aee8a7e09a05210cb491f9b/index.html#L61

https://github.com/brianeto/link-tutorial/blob/226d8c21bce472dc4aee8a7e09a05210cb491f9b/index.html#L70

https://github.com/brianeto/link-tutorial/blob/226d8c21bce472dc4aee8a7e09a05210cb491f9b/index.html#L77

https://github.com/brianeto/link-tutorial/blob/226d8c21bce472dc4aee8a7e09a05210cb491f9b/index.html#L84

https://github.com/brianeto/link-tutorial/blob/226d8c21bce472dc4aee8a7e09a05210cb491f9b/index.html#L113

https://github.com/brianeto/link-tutorial/blob/226d8c21bce472dc4aee8a7e09a05210cb491f9b/index.html#L118

https://github.com/brianeto/link-tutorial/blob/226d8c21bce472dc4aee8a7e09a05210cb491f9b/index.html#L125

https://github.com/brianeto/link-tutorial/blob/226d8c21bce472dc4aee8a7e09a05210cb491f9b/index.html#L132

https://github.com/brianeto/link-tutorial/blob/226d8c21bce472dc4aee8a7e09a05210cb491f9b/index.html#L141

https://github.com/brianeto/link-tutorial/blob/226d8c21bce472dc4aee8a7e09a05210cb491f9b/index.html#L148
